README.md

```diff
@@ -1 +1,394 @@
+
+<!-- > Describe the tech stack and include only the relevant sections that apply to your project. -->
+<details>
+  <summary>Client Side / Front-End</summary>
+  <ul>
+    <li><a href="https://www.w3.org/html/">HTML</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/">CSS</a></li>
+    <li><a href="https://www.javascript.com/">Javascript</a></li>
+    <li><a href="https://create-react-app.dev/">React</a></li>
+    <li><a href="https://reactrouter.com/">React Router</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server Side / Back-End</summary>
+  <ul>
+    <li><a href="https://www.json.org/json-en.html">JSON</a></li>
+    <li><a href="https://nodejs.org/en/">Node.js</a></li>
+    <li><a href="https://www.npmjs.com/package/big.js">big.js</a></li>
+    <li><a href="https://api-ninjas.com/api/quotes">API Ninjas</a></li>
+    <!-- <li><a href=""> - </a></li>
+    <li><a href=""> - </a></li> -->
+  </ul>
+</details>
+
+<details>
+    <summary>Package, Library, Framework</summary>
+      <ul>
+        <li><a href="https://jestjs.io/">Jest</a></li>
+      </ul>
+  </details>
+
+<!-- Features -->
+### Tools i have used for this project <a name="tools"></a>
+  <details>
+    <summary>Code Convention, Code Analysis</summary>
+      <ul>
+        <li><a href="https://eslint.org/">ESLint</a></li>
+        <li><a href="https://webhint.io/">Webhint</a></li>
+        <li><a href="https://stylelint.io/">Stylelint</a></li>
+        <li><a href="https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk?hl=en">Lighthouse</a></li>
+        <li><a href="https://www.npmjs.com/package/npm-check">node_modules checker</a></li>
+      </ul>
+  </details>
+  <details>
+    <summary>Version Control, CI/CD, Hosting Service</summary>
+      <ul>
+        <li><a href="https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow">GitFlow</a></li>
+        <li><a href="https://pages.github.com/">Github Pages</a></li>
+        <li><a href="https://github.com/features/actions">Github Actions</a></li>
+        <li><a href="https://git-scm.com/">Git</a></li>
+      </ul>
+  </details>
+  <details>
+    <summary>IDE, Desktop Apps, Other Tools</summary>
+      <ul>
+        <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+        <li><a href="https://desktop.github.com/">Github Desktop</a></li>
+        <!-- <li><a href="https://www.figma.com/">Figma</a></li> -->
+      </ul>
+  </details>
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+- Make simple calculations.
+- Read a random math-related quote.
+- single page apps (SPA).
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name=""></a>
+
+<!-- > Add a link to your deployed project. -->
+- Click the following url for [... Live Demo ...](https://math-magicians-rep-giuvx85y6-fickryiman.vercel.app/)
```

Comment:
The link to your deployed project is currently broken and this is giving your readme a less professional look, kindly fix this, or optionally, you can take away the live demo section since deployment is not required in this milestone👍.
<img width="818" alt="image" src="https://github.com/fickryiman/math-magicians-rep/assets/64643995/e03fa23c-13c3-46e4-ac4f-c822f38c0c22">